Packs/Armis/Integrations/ArmisEventCollector/ArmisEventCollector.yml

```diff
@@ -0,0 +1,71 @@
+category: Analytics & SIEM
+commonfields:
+  id: ArmisEventCollector
+  version: -1
+configuration:
+- defaultvalue: https://<armis-instance>.armis.com
+  display: Server URL
+  name: server_url
+  required: true
+  type: 0
+  section: Connect
+- displaypassword: API Secret Key
+  additionalinfo: The API Secret Key allows you to programmatically integrate with the Armis ecosystem.
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+  section: Connect
+- display: Number of events to fetch per type
+  name: max_fetch
+  additionalinfo: The maximum number of events to fetch per event type.
+  type: 0
+  defaultvalue: 1000
+  section: Collect
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  section: Connect
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  section: Connect
+- display: Event types to fetch
+  name: event_types_to_fetch
+  section: Collect
+  required: true
+  type: 16
+  defaultvalue: Alerts,Threat activities
+  options:
+  - Alerts
+  - Threat activities
+description: 'Collects alerts & threat activities from Armis resources.'
+display: Armis Event Collector
+name: ArmisEventCollector
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: 'false'
+      description: Set this argument to true in order to create events, otherwise the command will only display them.
+      name: should_push_events
+      predefined:
+      - 'true'
+      - 'false'
+      required: true
+    - description: The date from which to fetch events. the format should be YYYY-MM-DD or YYYY-MM-DDT:HH:MM:SS, if not specified, the current date will be used.
+      name: from_date
+      required: false
+    description: Manual command to fetch and display events. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to events duplication and API request limitation exceeding.
```

Comment:
```suggestion
    description: Manual command to fetch and display events. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to events duplication and exceeding the API request limitation.
```